int32 i
int32 j
bool grab
std_msgs/String robot_name
std_msgs/String link_name